certified-connectors/Seismic Engagement/apiDefinition.swagger.json

```diff
@@ -0,0 +1,893 @@
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Seismic.LiveSend.LiveSendLinkContentBasicInfo": {
+      "description": "LiveSend link content Basic Info.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Content Id",
+          "type": "string",
+          "x-ms-summary": "Id"
+        },
+        "versionId": {
+          "description": "Content Version Id",
+          "type": "string",
+          "x-ms-summary": "Version Id"
+        },
+        "teamsiteId": {
+          "description": "TeamSiteId",
+          "type": "string",
+          "x-ms-summary": "TeamSite Id"
+        }
+      }
+    },
+    "Seismic.LiveSend.LiveSendLinkResponse": {
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "The LiveSend Link URL to be returned to the caller.",
+          "type": "string"
+        },
+        "id": {
+          "description": "Unique ID for the LiveSend link.",
+          "type": "string"
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "settings": {
+          "$ref": "#/definitions/Seismic.LiveSend.LiveSendLinkSettings"
+        },
+        "content": {
+          "description": "The contents for this LiveSend link.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Seismic.LiveSend.LiveSendLinkContent"
+          }
+        }
+      }
+    },
+    "Seismic.LiveSend.LiveSendLinkSettings": {
+      "description": "LiveSend link settings",
+      "type": "object",
+      "properties": {
+        "expiresAt": {
+          "format": "date-time",
+          "x-ms-summary": "Expires At",
+          "description": "Expiration date for the LiveSend link.",
+          "type": "string"
+        },
+        "password": {
+          "description": "Password to access the LiveSend link content.",
+          "type": "string",
+          "x-ms-summary": "Password"
+        },
+        "allowDownload": {
+          "description": "Can the viewer download the content?",
+          "type": "boolean",
+          "x-ms-summary": "Allow Download"
+        },
+        "notificationType": {
+          "description": "Notification type for the sender. Valid values are 'All','Session', and 'None'.",
+          "type": "string",
+          "x-ms-summary": "Notification Type"
+        },
+        "singleView": {
+          "description": "Can the LiveSend link only be viewed once?",
+          "type": "boolean",
+          "x-ms-summary": "Single View"
+        }
+      }
+    },
+    "Seismic.LiveSend.LiveSendPasswordSettings": {
+      "type": "object",
+      "properties": {
+        "requiresLowerCase": {
+          "description": "Password requires attleast one lowercase letter",
+          "type": "boolean"
+        },
+        "requiresNumber": {
+          "description": "Password requires atleast one number",
```

Comment:
@rkushvaha all files should be free of grammatical and spelling errors. here in the description you have "atleast" may I ask you please to change to "at least"

Reply:
corrected